chromium_src/content/public/browser/url_data_source.cc

```diff
@@ -18,7 +18,7 @@ std::string URLDataSource::GetContentSecurityPolicy(
     case network::mojom::CSPDirectiveName::ScriptSrc:
       // Note: Do not add 'unsafe-eval' here. Instead override CSP for the
       // specific pages that need it, see context http://crbug.com/525224.
-      return "script-src chrome://resources chrome://brave-resources 'self';";
+      return "script-src chrome://resources 'self';";
```

Comment:
I don't think this is needed anymore and is actually probably unsafe because it doesn't call IsChromeUntrustedDataSource
The entire purpose of this was to add `chrome://brave-resources` so we should remove it

Reply:
Fixed in dcb57714ebeeb1c7854d3f99b3290f0c5367bd99.